overlord/ifacestate/ifacemgr.go

```diff
@@ -73,54 +73,116 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, extra []interfa
 	return m, nil
 }
 
+func initialConnectAttributes(s *state.State, plugSnap string, plugName string, slotSnap string, slotName string) (map[string]interface{}, error) {
+	// Combine attributes from plug and slot, store them in connect task.
```

Comment:
This piece is bogus. Plugs and slots have their own attributes. We don't want a single bag with both. Let's have a call to discuss this with more bandwidth.